docs/06-hardware/cases.md

```diff
@@ -79,3 +79,26 @@ _Lots of space for drives, even with poor cable management._
 * Pricing - I purchased mine on promotion for about $80 years back, but they seem to have nearly doubled in price and sit at around $140 new.
 
 While this chassis is larger than other offerings, I chose it for its generous number of 3.5" bays, included quality fans and support for ATX power supplies. Front I/O is adequate, and the blue power LED can simply be unplugged should it cause a disturbance. Included are two fans that are controlled by a built-in 3 position switch. I've always kept it at its lowest and temperature has never been a concern.
+
+## NZXT Phantom 240
+
+> Review contributed by [Vogete](https://github.com/vogete)
+
+The NZXT Phantom 240 might not stand out from the croud in any of the features of aesthetics it offers (though subjectically it looks amazing in white with its stormtrooper style), but it will definitely impress you with its price. At just $69.99 (release price), it's hard to beat the value of the materials you get with it. Unfortuantely as far as I can tell it's not being sold much at most places, so it might be hard, or impossible to come by.
```

Comment:
Spell check "unfortunately".

Reply:
I spell checked everything now and fixed all my mistakes